node-notes-cli/updateItem.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+const updateItem = (inputedItem1, inputedItem2) => {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) throw err;
+    data = JSON.parse(data);
+    const { notes } = data;
+    for (const i in notes) {
+      if (i === inputedItem1) {
+        notes[i] = inputedItem2;
+      }
+    }
```

Comment:
Since you know which note id to target, you can update it directly without a loop.

```js
notes[inputedItem1] = inputedItem2;
```